lib/menu.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'takeaway'
+#require_relative 'order'
+
+module Menu
+
+  MENU = {"Chicken" => 5.00, "Madras" => 6.00, "Nan Bread" => 2.50, "Cobra" => 4.00}
+
+  def show_menu
+    MENU.each { |item, price| "#{item}, #{price}"}
+  end
+
+  def hash_menu
```

Comment:
The responsibility of `Menu` should be to provide an interface for listing dishes and prices without exposing its internal implementation (i.e. it should not return the internal hash)